sklearn/model_selection/_search.py

```diff
@@ -355,7 +360,7 @@ def __repr__(self):
         """Simple custom repr to summarize the main info"""
         return "mean: {0:.5f}, std: {1:.5f}, params: {2}".format(
             self.mean_validation_score,
-            np.std(self.cv_validation_scores),
+            self.corrected_std,
```

Comment:
Oh yeah, sorry. But the change in printing should be done in the old module as well.